modules/admin_manual/pages/maintenance/backup_and_restore/restore.adoc

```diff
@@ -4,49 +4,77 @@
 
 == Introduction
 
-When you install ownCloud from packages, follow these steps to restore
-your ownCloud installation. Start with a fresh ownCloud package
-installation in a new, empty directory. Then restore these items from
-your xref:maintenance/backup_and_restore/backup.adoc[Backup]:
+Depending how the ownCloud instance has been installed, you may need slightly different steps to restore it from a backup.
 
-1.  Your `config/` directory.
-2.  Your `data/` directory.
-3.  Your ownCloud database.
-4.  Your custom theme files, if you have any. (See xref:developer_manual:core/theming.adoc[Theming ownCloud])
+In any case you need the following components from your backup:
 
-If you install ownCloud from the source tarballs, you may safely
-restore your entire ownCloud installation from backup, with the
-exception of your ownCloud database. Databases cannot be copied, instead you
-must use the database tools to make a correct restoration.
+.  The `config/` directory.
+.  The `data/` directory. +
+Note that the `data/` directory may not only contain user files, but also keys for encryption.
+.  The `apps/` directory. +
+Note that this is only necessary if you are not using the `apps-external/` directory and have added own apps or themes.
+.  The `apps-external/` directory. +
+Note that this is only necessary if it exists and is in use.
+.  The ownCloud database.
+.  The custom theme files, if you had any. See xref:developer_manual:core/theming.adoc[Theming ownCloud]. +
+Note that theme files are usually located in either the `apps/` or `apps-external/` directory.
 
-After you have completed the restoration, see how to xref:installation/manual_installation/manual_installation.adoc#script-guided-installation[Set Correct Permissions].
+IMPORTANT: If you have customized user home directories or a custom location for encryption keys, you have to manually take care backing them up and restoring them to the same location.
 
-== Restore Directories
+== Prerequisites
 
-Simply copy your `config/` and `data/` folders to your ownCloud
-environment, for example by running the following command from the backup directory:
+To ensure secure your restore process, stop your webserver to prevent users trying to access ownCloud via the web. As an alternative, you can stop serving the virtual host for ownCloud:
```

Comment:
```suggestion
To ensure a secure restore process, stop your web server to prevent users from accessing ownCloud via the web. As an alternative, you can stop serving the virtual host for ownCloud:
```